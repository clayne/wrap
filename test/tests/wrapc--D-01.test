wrapc | /dev/null | | hello.d | 0
