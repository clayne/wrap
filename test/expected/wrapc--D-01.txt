/+
 + The D programming language is an object-oriented, imperative, multi-paradigm
 + system programming language created by Walter Bright of Digital Mars and
 + released in 2001.
 +/
void main() {
  writeln( "hello, world" );
}
